plugins/catalog-backend-module-ldap/src/ldap/config.ts

```diff
@@ -311,8 +323,21 @@ function readGroupMapConfig(
 }
 
 function readUserConfig(
-  c: Config,
+  c: Config | Config[] | undefined,
 ): RecursivePartial<LdapProviderConfig['users']> {
+  if (!c) {
+    return undefined;
+  }
+  if (Array.isArray(c)) {
+    return c.map(it => {
+      return {
+        dn: it.getString('dn'),
```

Comment:
would be unfortunate to duplicate code like this, if you normalize the input to an array as mentioned elsewhere this will also get rid of its repetition

Reply:
extracted in separate function, but this is actually where the normalization is happening atm.